src/components/BreedPhotos/BreedPhotos.tsx

```diff
@@ -0,0 +1,94 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import ReactPaginate from 'react-paginate';
+import ImageCard from '../ImageCard/ImageCard';
+import formatBreedName from '../../helpers/formatName';
+import { BreedPhotosGrid, StyledMain, StyledPaginateContainer } from './styles';
+
+type BreedPhotosProps = {
+    breedName: string;
+    itemsPerPage: number;
+};
+
+type ItemProps = {
+    currentItems: Array<string>;
+}
+
+function Items({ currentItems }: ItemProps) {
+    return (
+      <>
+        {currentItems &&
+          currentItems.map((item) => (
+            <ImageCard key={item} imageUrl={item} />
+          ))}
+      </>
+    );
+  }
+
+export default function BreedPhotos({ breedName, itemsPerPage }: BreedPhotosProps) {
+    // Set empty list of items.
```

Comment:
Upon entering this component, the network keeps calling the `https://dog.ceo/api/breed/${breedName}/images` api non stop! Do you think we can avoid this ?